component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroSchema.java

```diff
@@ -39,13 +42,19 @@
 @ToString(of = "delegate")
 public class AvroSchema implements org.talend.sdk.component.api.record.Schema, AvroPropertyMapper, Unwrappable {
 
+    // alias that indicate field is metadata if present.
+    static final String METADATA_ALIAS_NAME = "__METADATA__";
```

Comment:
Use the already existing `KeysForAvroProperty` and prefix this constant with a `talend.xxx` namespace.

Reply:
I didn't know this interface.
Done